src/utils/mediaSource.ts

```diff
@@ -0,0 +1,10 @@
+import { MediaSourceInfo } from '@jellyfin/sdk/lib/generated-client';
+
+/**
+ * Checks if the media source is an HLS stream.
+ * @param mediaSource The media source.
+ * @returns _true_ if the media source is an HLS stream, _false_ otherwise.
+ */
+export function isHls(mediaSource: MediaSourceInfo|null|undefined): boolean {
+    return (mediaSource?.TranscodingSubProtocol || mediaSource?.Container) === 'hls';
```

Comment:
Hmm, https://github.com/jellyfin/jellyfin/pull/10153 will change the type of `TranscodingSubProtocol`.

Reply:
I think it will be fine as long as `hls` matches the enum value.